### ISSUE TITLE
[iceprod2] dataset common name (Trac #1682)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1674]
On submission, set the "dataset name" - a common name for the dataset that is easier to remember than the dataset id.  This should be unique, and available from the job as $(dataset).  The previous "dataset" value should be moved to $(dataset_id).

Example: an incrementing number, starting at 20000.

Migrated from https://code.icecube.wisc.edu/ticket/1682
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "On submission, set the \"dataset name\" - a common name for the dataset that is easier to remember than the dataset id.  This should be unique, and available from the job as $(dataset).  The previous \"dataset\" value should be moved to $(dataset_id).\n\nExample: an incrementing number, starting at 20000.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] dataset common name", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-05-02T20:54:02", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160504T16:42:16: 
* david.schultz changed status from "new" to "closed"
* david.schultz commented:

> In changeset 145488:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="145488"
> add numeric dataset name. closes #1674
> ```


* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
